src/ripple/protocol/impl/STTx.cpp

```diff
@@ -517,4 +487,29 @@ sterilize (STTx const& stx)
     return std::make_shared<STTx const>(std::ref(sit));
 }
 
+STTx
+assemble_tx (
```

Comment:
Isn't a design that offers a constructor better than one which forces you to use a free function?